.github/workflows/ci.yml

```
@@ -23,6 +23,18 @@ jobs:
                 curl -L -O https://gitlab.tiker.net/inducer/ci-support/raw/main/prepare-and-run-flake8.sh
                 . ./prepare-and-run-flake8.sh "$(basename $GITHUB_REPOSITORY)" test examples benchmarks
 
+    pylint:
+        name: Pylint
+        runs-on: ubuntu-latest
+        steps:
+        -   uses: actions/checkout@v2
+        -   name: "Main Script"
+            run: |
+                USE_CONDA_BUILD=1
+                EXTRA_INSTALL="pyvisfile scipy matplotlib"
+                curl -L -O https://gitlab.tiker.net/inducer/ci-support/raw/main/prepare-and-run-pylint.sh
+                . ./prepare-and-run-pylint.sh "$(basename $GITHUB_REPOSITORY)" test/test_*.py
```

```suggestion
                curl -L -O https://tiker.net/ci-support-v0
                . ci-support-v0
                build_py_project
                run_pylint "$(basename $GITHUB_REPOSITORY)" test/test_*.py
```
(also on the Gitlab side)

---

Fixed in 
https://github.com/inducer/sumpy/compare/6be06e736169d8ea8f85fcef3c16a31b5d793415..e5ed7b5666c6b6b5ab3cfe838ac1f004bec4ef35

I copied this from `boxtree`, so probably other projects need updating as well.

---

They do need updating, but there's no huge rush IMO. More important to make things better over time.